locales/en-US/app.ftl

```diff
@@ -54,12 +54,43 @@ CompareHome--form-label-profile2 = Profile 2:
 CompareHome--submit-button =
     .value = Retrieve profiles
 
+## DebugWarning
+
+DebugWarning--warning-message =
+    .message =
+    This profile was recorded in a build without release optimizations.
+    Performance observations might not apply to the release population.
+
+## Details
+
+Details--open-sidebar-button =
+    .title = Open the sidebar
+Details--close-sidebar-button =
+    .title = Close the sidebar
+Details--error-boundary-message =
+    .message = Uh oh, some unknown error happened in this panel.
+
 ## FooterLinks
 
 FooterLinks--legal = Legal
 FooterLinks--Privacy = Privacy
 FooterLinks--Cookies = Cookies
 
+## FullTimeline
+
+FullTimeline--graph-type = Graph type:
+FullTimeline--categories-with-cpu = Categories with CPU
+FullTimeline--categories = Categories
+FullTimeline--stack-height = Stack height
+
+# This string is used as the text of the track selection button.
+# Displays the ratio of visible tracks count to total tracks count in the timeline.
+# Variables:
+#   $visibleTrackCount (Number) - Visible track count in the timeline
+#   $totalTrackCount (Number) - Total track count in the timeline
+FullTimeline--tracks-visible =
+    <span>{ $visibleTrackCount }</span> / <span>{ $totalTrackCount }</span> tracks visible
```

Comment:
Should we have 2 different strings depending on the value for `$visibleTrackCount`?

Reply:
This results in `1/2 tracks visible`, right? Then, no.

A locale can still make this a plural form, if they need to, e.g. (`1 of 2 tracks visible`, with plural depending on the `2`).

Reply:
I would think that plural depends on 1 ?
1 track is visible out of 2 tracks

Reply:
I think it depends on how you form the sentence. You can also say "1 out of 2 tracks is visible". This sounds more natural to me, but I'm also not really great at this 😄
But nevertheless, locales can still make a plural form and decide whether to look at `visibleTrackCount` or `totalTrackCount` themselves afaik.

Reply:
Correct, it depends on the form, and locales can pick whatever works for them..

And in your example, the plural still depends on 2
1 track is visible out of 1 track

Reply:
Cool, then I'm going to land it this way and let the other locales decide their own. Thanks!

Reply:
Sorry to come back at this :-) I know this is only for the english translation, obviously other translations will do as they wish.

So should that be `1 / 2 track visible` or `1 / 2 tracks visible`?
What about `1 / 1 track visible` or `1 / 1 tracks visible`? If the former, should we depend on `$totalTrackCount`?

Would that be different with `1 / 2 visible track` vs `1 / 2 visible tracks`?

Thanks :-)